rss.py

```
@@ -0,0 +1,97 @@
+
+    if any(keyword in link_lower for keyword in link_blacklist):
+        return False
+    if any(keyword in title_lower for keyword in title_blacklist):
+        return False
+    return True
+
+def download_comic(comic_info: dict) -> None:
+    url = comic_info.get("link")
+    headers = {"User-Agent": "Mozzilla/5.0"}
+
+    reponse = requests.get(url, headers)
```

Ruff (0.11.9)

Probable use of `requests` call without timeout

(S113)